lib/mayaUsd/utils/utilSerialization.cpp

```diff
@@ -137,6 +137,9 @@ void updateAllCachedStageWithLayer(SdfLayerRefPtr originalLayer, const std::stri
         std::vector<UsdStageRefPtr> stages = cache.FindAllMatching(originalLayer);
         for (const auto& stage : stages) {
             auto sessionLayer = stage->GetSessionLayer();
+            // Note: See comments in lib\mayaUsd\nodes\proxyShapeBase.cpp, line 823
```

Comment:
The line number is most likely to go out of sync. Perhaps we could refer to the comment by function name ?